app/views/expenses/index.html.erb

```diff
@@ -1,19 +1,18 @@
 <p style="color: green"><%= notice %></p>
- <div class="container mx-auto flex justify-start items-center bg-primary text-white font-light px-3 py-4 exp-text">
-    <%= link_to categories_path do %>
-      <i class="fa-solid fa-arrow-left fa-xl"></i>
-    <% end %>
-    <h2>Expenses for <%= @category.name %></h2>
-</div>
-<div class="total-amount">
-  <p>Total Amount: <%= number_to_currency(@category.expenses.sum(:amount)) %></p>
+
+<div class="container mx-auto flex justify-start items-center bg-primary text-white font-light px-3 py-4 exp-text">
+  <%= link_to categories_path do %>
+    <i class="fa-solid fa-arrow-left fa-xl"></i>
+  <% end %>
+  <h5 class="font-light grow text-center">Expenses for <%= @category.name %></h5>
 </div>
 
 <div id="payments">
-<% @category.expenses.each do |expense| %>
-  <%= render expense %>
-<% end %>
+  <% @category.expenses.each do |expense| %>
+    <%= render expense %>
+  <% end %>
 </div>
 
-
-
+<div class="total-amount">
+  <p>Total Amount: <%= number_to_currency(@category.expenses.sum(:amount)) %></p>
+</div>
```

Comment:
- [ ] As mentioned by the previous reviewer I highly recommend making the amount of expense right after the navbar to match the given design so kindly make it As per the design, your total amount for each expenses should be visible on the top of the screen just after the navigation as per the Behance design provided to you. Right now it is visible at the bottom, after all expenses which is not good at all, because as the list of expenses grows your total amount will be visible at the very bottom and the user needs to scroll down until he/she doesn't get the total amount container, which again results in bad user experience. 🔥🔥🙌🙌

- Your design 
<img width="290" alt="image" src="https://github.com/hydaline-code/expense_tracker/assets/58553711/ee31eeab-6bc9-4ead-86f9-d18e993cba1e">

- Given
<img width="173" alt="image" src="https://github.com/hydaline-code/expense_tracker/assets/58553711/f42a81d1-a35a-4ffa-89b9-6e3775fe8f5b">